jose-jwt/jose-jwt.csproj

```diff
@@ -144,4 +145,8 @@ RFC 7797:
       <Compile Remove="RsaKey.cs" />
     </ItemGroup>
 
+    <ItemGroup>

```

Comment:
why we do we need it?

Reply:
- System.ValueTuple... I return a tuple from JWE.Decrypt.. you get the plaintext, Jose headers (and aad if present),
- On net461 and netstandard1.4 need to include this as a project dependency. For netstandard2.1 its not needed.
- I've cleaned the csproj to only include for those targets now and committed the change.